oktapam/convert/service_account_checkout_settings.go

```diff
@@ -0,0 +1,195 @@
+package convert
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/int32validator"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/descriptions"
+
+	"github.com/atko-pam/pam-sdk-go/client/pam"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// ServiceAccountCheckoutSettingsModel represents the Terraform model for service account checkout settings
+type ServiceAccountCheckoutSettingsModel struct {
+	CheckoutRequired          types.Bool  `tfsdk:"checkout_required"`
+	CheckoutDurationInSeconds types.Int32 `tfsdk:"checkout_duration_in_seconds"`
+	IncludeList               types.List  `tfsdk:"include_list"`
+	ExcludeList               types.List  `tfsdk:"exclude_list"`
+}
+
+// ServiceAccountSettingNameObjectModel represents the Terraform model for service account setting name object
+type ServiceAccountSettingNameObjectModel struct {
```

Comment:
Do we really have a thing called "name object" and "named object" and they're different?